apps/gitness/src/pages-v2/repo/repo-commit-details.tsx

```diff
@@ -34,7 +34,6 @@ export default function RepoCommitDetailsPage({ showSidebar = true }: { showSide
         routes.toPullRequest({ spaceId, repoId, pullRequestId: pullRequestId.toString() })
       }
       useCommitDetailsStore={useCommitDetailsStore}
-      showSidebar={showSidebar}
```

Comment:
are we going to show sidebar all the time?

Reply:
Files sidebar is passed via Outlet anyway. This prop was only changing the layout, which is unnecessary after my refactoring of the `RepoCommitDetailsView`
@praneshg239